src/lib/test/InfoTests.cpp

```diff
@@ -207,28 +173,33 @@ void InfoTests::testGetSlotInfoAlt()
 	// (Re)initialize the token
 	rv = C_Initialize(NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_OK);
-	rv = C_InitToken(SLOT_INIT_TOKEN, pin, pinLength, label);
+	rv = C_InitToken(m_initializedTokenSlotID, m_soPin1, m_soPin1Length, label);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 	C_Finalize(NULL_PTR);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, &slotInfo);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_CRYPTOKI_NOT_INITIALIZED);
 
 	rv = C_Initialize(NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, NULL_PTR);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, NULL_PTR);
 	CPPUNIT_ASSERT(rv == CKR_ARGUMENTS_BAD);
 
-	rv = C_GetSlotInfo(SLOT_INVALID, &slotInfo);
+	rv = C_GetSlotInfo(m_invalidSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_SLOT_ID_INVALID);
 
-	rv = C_GetSlotInfo(SLOT_NO_INIT_TOKEN, &slotInfo);
+	rv = C_GetSlotInfo(m_notInitializedTokenSlotID, &slotInfo);
 	CPPUNIT_ASSERT(rv == CKR_OK);
 	CPPUNIT_ASSERT((slotInfo.flags & CKF_TOKEN_PRESENT ) == CKF_TOKEN_PRESENT);
 	CPPUNIT_ASSERT((slotInfo.flags & CKF_REMOVABLE_DEVICE ) == CKF_REMOVABLE_DEVICE);
 
 	C_Finalize(NULL_PTR);
+#ifndef _WIN32
```

Comment:
No this is intentional.
Before it was a setUp() method in each test class. All these methods was setting the "SOFTHSM2_CONF" to the default "./softhsm2.conf". This was done before each test since setUp() is executed before each test.
Now there are no setUp() in any of the test classes. Instead each test class inherits the setUp() from any of 2 base classes (TestsBase or TestsNoPINInitBase). Instead of doing the setting in TestsNoPINInitBase::setUp() that is executed before each test the setting of the default is only done twice:

- in p11test.cpp before start of the testing.

- in this method after an alternative has been tested. There is a setting of an alternative to be tested in the beginning of this method. Here it is set back to the default.